airflow/providers/google/cloud/transfers/facebook_ads_to_gcs.py

```diff
@@ -85,7 +85,6 @@ def __init__(
         bucket_name: str,
         object_name: str,
         fields: List[str],
-        params: Dict[str, Any],
```

Comment:
I _suspect_ that this params was meant to be different to the params from the base operator and this is a name collision (that until now hasn't caused any problems)